src/file.toit

```diff
@@ -5,8 +5,11 @@
 import reader show Reader
 import writer show Writer
 
-// Manipulation of files on a filesystem.  Currently not available on embedded
-// targets.  Names work best when imported without "show *".
+import system
+import .directory
+
+// Manipulation of files on a filesystem.
+// Names work best when imported without "show *".
```

Comment:
I don't see why that's the case.
I typically use the `file` library with `file.read-content` ...

Reply:
I think that fits with what it is saying.